src/gwt/src/org/rstudio/studio/client/workbench/prefs/views/AppearancePreferencesPane.java

```diff
@@ -32,11 +28,14 @@
 import org.rstudio.core.client.StringUtil;
 import org.rstudio.core.client.resources.ImageResource2x;
 import org.rstudio.core.client.theme.ThemeFonts;
-import org.rstudio.core.client.widget.SelectWidget;
+import org.rstudio.core.client.widget.*;
```

Comment:
I've avoided the wildcard `imports`, but only because I hadn't noticed any before IntelliJ started doing them. I don't know of any actual problem doing it with the wildcards.

Reply:
Oh, interesting I didn't actually realize that change had been made. They're generally not recommended because they can clutter your namespace.